.gitignore

```
@@ -8,3 +8,4 @@
 /spec/reports/
 /tmp/
 /nzbn-0.1.0.gem
+/.idea/
```

```suggestion
/.idea/

```
Add a new line to the end of this file